pkgs/tools/misc/cyberchef/default.nix

```diff
@@ -0,0 +1,77 @@
+{ lib, pkgs, fetchFromGitHub, stdenv, ... }:
+
```

Comment:
Don't use pkgs as an input.

Reply:
I don't think it's possible to use node2nix without giving it pkgs.